docs/docs/overview/quickstart/get-started-with-vanilla-js.md

```diff
@@ -0,0 +1,315 @@
+
+You can find out more about the web component [here](https://docs.novu.co/notification-center/web-component/).
+
+In our case, we'll be using the bundled version of the Notification Center Web Component that is available on the CDN.
+
+You can load the Notification center component using the CDN as follows:
+
+```html
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    ...
+    <!-- CDN link for Novu notification center component -->
+    <script src="https://novu-web-component.netlify.app/index.js"></script>
+  </head>
+  <body>
+    <!-- HTML body -->
+    <!-- the notification center -->
+    <notification-center-component
+      style="display: inline-flex"
+      application-identifier="<YOUR_APPLICATION_IDENTIFIER>"
+      subscriber-id="<YOUR_SUBSCRIBER_ID>"
+    ></notification-center-component>
+  </body>
+</html>
+```
+
+You'll notice the use of two things in the above code - `application-identifier` and `subscriber-id`.
+
+Application identifier is a public key used to identify your application. You can get your own application identifier from the [Novu dashboard settings](https://web.novu.co/settings).
+
+And subscribers are users to which notifications will be sent. They are identified by a `subscriberID` which you can also find in the [Novu subscribers dashboard](https://web.novu.co/subscribers). Let's learn more about subscribers.
+
+## Subscribers in Novu
+
+If you click “Subscriber” on the left sidebar of [Novu dashboard](https://web.novu.co/subscribers), you’ll see the subscriber list. By default, there will only be one subscriber as you’re automatically added as a subscriber when you sign up for Novu:
+
+![Subscribers page on the Novu dashboard](https://res.cloudinary.com/dxc6bnman/image/upload/v1685465978/guides/subscribers_1_opmw0s.png)
+
+Now, let’s create a subscriber on Novu. After creating a subscriber, we’ll trigger notification to this subscriber. Subscribers are identified by a unique `subscriberId`.
+
+With Novu, you can create a subscriber using any of its SDKs (Node.js, PHP, .NET, Go, Ruby, Python and Kotlin). The code to create a subscriber in Novu is:
+
+<Tabs groupId="language" queryString>
+
+  <TabItem value="js" label="Node.js">
+
+```javascript
+import { Novu } from '@novu/node';
+
+const novu = new Novu('<YOUR_NOVU_API_KEY>');
+
+await novu.subscribers.identify('123', {
+  firstName: 'Sumit',
+  lastName: 'Saurabh',
+  returnUser: true,
+});
+```
+
+  </TabItem>
+  <TabItem value="php" label="PHP">
+
+```php
+use Novu\SDK\Novu;
+
+$novu = new Novu('<NOVU_API_KEY>');
+
+$novu->createSubscriber([
+    'subscriberId' => '123',
+    'firstName' => 'Sumit',
+    'lastName' => 'Saurabh',
+    'phone' => '+13603963366',
+    'avatar' => 'https://example.com/images/avatar.jpg',
+    'locale' => 'en',
+    'data' => [
+      'customKey1' => 'customVal1',
+      'customKey2' => 'customVal2'
+    ]
+]);
```

Comment:
Like I mentioned in the other guide, the data needs to be similar/thesame in both code samples

Reply:
done ✅